src/Serialize.Linq.NetStandard16/Serialize.Linq.NetStandard16.csproj

```diff
@@ -1,18 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netstandard1.6</TargetFramework>
     <RootNamespace>Serialize.Linq</RootNamespace>
-    <Version>1.4.0</Version>
+    <Version>1.4.0$(VersionSuffix)</Version>
```

Comment:
necessary for prerelease versioning

Reply:
but also versionsuffix is by default empty, so doesn't affect current versioning scheme